### PR TITLE
Added closing / to XML elements (bsc#1206096)

### PR DESCRIPTION
**Pull request by tbazant**
### PR creator: Description

XML configuration examples missed closing slash, this update fixes that

### PR creator: Are there any relevant issues/feature requests?

* bsc#1206069

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP4/openSUSE Leap 15.4
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 SP0
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/ay_partitioning.xml [hunk 1]
```diff
@@ -97,11 +97,11 @@
 <screen>&lt;general&gt;
    &lt;storage&gt;
      &lt;proposal&gt;
-       &lt;lvm config:type="boolean"&gt;true&lt;lvm&gt;
-       &lt;windows_delete_mode config:type="symbol"&gt;all&lt;windows_delete_mode&gt;
+       &lt;lvm config:type="boolean"&gt;true&lt;lvm/&gt;
+       &lt;windows_delete_mode config:type="symbol"&gt;all&lt;windows_delete_mode/&gt;
```
Comment (tomschr) [inline xml/ay_partitioning.xml:101]:
```suggestion
       &lt;lvm config:type="boolean">true&lt;/lvm>
       &lt;windows_delete_mode config:type="symbol">all&lt;/windows_delete_mode>
```

File: xml/ay_partitioning.xml [hunk 2]
```diff
@@ -97,11 +97,11 @@
...
      &lt;/proposal&gt;
-   &lt;storage&gt;
-&lt;general&gt;</screen>
+   &lt;/storage&gt;
+&lt;/general&gt;</screen>
```
Comment (tomschr) [inline xml/ay_partitioning.xml:104]:
```suggestion
   &lt;/storage>
&lt;/general></screen>
```

File: xml/ay_partitioning.xml [hunk 3]
```diff
@@ -121,7 +121,7 @@
       When set to <literal>true</literal>, &ay; resizes Windows partitions if
       needed to make room for the installation.
      </para>
-<screen>&lt;resize_windows config:type="boolean"&gt;false&lt;/resize_windows&gt;</screen>
+<screen>&lt;resize_windows config:type="boolean"&gt;false&lt;/resize_windows/&gt;</screen>
```
Comment (tomschr) [inline xml/ay_partitioning.xml:124]:
```suggestion
<screen>&lt;resize_windows config:type="boolean">false&lt;/resize_windows></screen>
```

File: xml/ay_partitioning.xml [hunk 4]
```diff
@@ -111,7 +111,7 @@
      <para>
       Creates an LVM-based proposal. The default is <literal>false</literal>.
      </para>
-<screen>&lt;lvm config:type="boolean"&gt;true&lt;/lvm&gt;</screen>
+<screen>&lt;lvm config:type="boolean"&gt;true&lt;/lvm/&gt;</screen>
```
Comment (tomschr) [inline xml/ay_partitioning.xml:114]:
```suggestion
<screen>&lt;lvm config:type="boolean">true&lt;/lvm></screen>
```

File: xml/ay_partitioning.xml [hunk 5]
```diff
@@ -144,7 +144,7 @@
        </para>
       </listitem>
      </itemizedlist>
-<screen>&lt;windows_delete_mode config:type="symbol"&gt;ondemand&lt;/windows_delete_mode&gt;</screen>
+<screen>&lt;windows_delete_mode config:type="symbol"&gt;ondemand&lt;/windows_delete_mode/&gt;</screen>
```
Comment (tomschr) [inline xml/ay_partitioning.xml:147]:
```suggestion
<screen>&lt;windows_delete_mode config:type="symbol">ondemand&lt;/windows_delete_mode></screen>
```

File: xml/ay_partitioning.xml [hunk 6]
```diff
@@ -167,7 +167,7 @@
        </para>
       </listitem>
      </itemizedlist>
-<screen>&lt;linux_delete_mode config:type="symbol"&gt;ondemand&lt;/linux_delete_mode&gt;</screen>
+<screen>&lt;linux_delete_mode config:type="symbol"&gt;ondemand&lt;/linux_delete_mode/&gt;</screen>
```
Comment (tomschr) [inline xml/ay_partitioning.xml:170]:
```suggestion
<screen>&lt;linux_delete_mode config:type="symbol">ondemand&lt;/linux_delete_mode></screen>
```

File: xml/ay_partitioning.xml [hunk 7]
```diff
@@ -190,7 +190,7 @@
        </para>
       </listitem>
      </itemizedlist>
-<screen>&lt;other_delete_mode config:type="symbol"&gt;ondemand&lt;/other_delete_mode&gt;</screen>
+<screen>&lt;other_delete_mode config:type="symbol"&gt;ondemand&lt;/other_delete_mode/&gt;</screen>
```
Comment (tomschr) [inline xml/ay_partitioning.xml:193]:
```suggestion
<screen>&lt;other_delete_mode config:type="symbol"&gt;ondemand&lt;/other_delete_mode></screen>
```

File: xml/ay_partitioning.xml [hunk 8]
```diff
@@ -200,7 +200,7 @@
       Enables encryption using the specified password. By default, encryption
       is disabled.
      </para>
-<screen>&lt;encryption_password&gt;some-secret&lt;/encryption_password&gt;</screen>
+<screen>&lt;encryption_password&gt;some-secret&lt;/encryption_password/&gt;</screen>
```
Comment (tomschr) [inline xml/ay_partitioning.xml:203]:
```suggestion
<screen>&lt;encryption_password&gt;some-secret&lt;/encryption_password></screen>
```

**Review by tomschr — CHANGES_REQUESTED** [review]
Hi @tbazant :wave: 

you wrote:

> XML configuration examples missed closing slash, this update fixes that

I'm sorry, but I think this is wrong. :slightly_smiling_face: It was not valid XML code.

I have two issues which I corrected in my suggestions below:

1. The use of `&gt;`.
    In XML, it's only required to mask the `<` character, but it's optional for `>`. If you are over this part, maybe it would be a good time to just use `>` instead of the entity. I think, `>` is more readable than `&gt;`.

2. The use of slashes.
    If you look in the XML code, you have something like this:
    
        &lt;windows_delete_mode config:type="symbol"&gt;all&lt;windows_delete_mode/&gt;
        
    This looks quite complicated, but let's replace the `&gt;` entity with `>` first:
    
        &lt;windows_delete_mode config:type="symbol">all&lt;windows_delete_mode/>

   Hope you can see it, but `<windows_delete_mode/>` would be an _empty_ element. However, to me, in that place it looks wrong and you probably mean an *end tag* to close the start tag. That end tag is `</windows_delete_mode>` (watch for the slash).

This means, all your slashes are not really required... :slightly_smiling_face: I corrected the XML fragments. Is that okay?

**Review by tbazant — COMMENTED** [review]
Thanks a lot @tomschr , i must have been half-blind in the morning :-)

**Pull request merged**